### PR TITLE
Update kubernetes.rst

### PR DESCRIPTION
**Pull request by karynzv**
edited the CrateDB version mentioned here which was not uniform. Replaced all version references for 5.0.3

## Summary of the changes / Why this is an improvement
Previously the versions mentioned here were not uniform, some places said 3.0.5 others had 4.1.2. Replaced them all for 5.0.3.

## Checklist

 - [ ] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

**Comment by seut** [comment]
Good catch! Maybe use most recent v5.1.1 instead?

File: docs/containers/kubernetes/kubernetes.rst [hunk 1]
```diff
@@ -228,8 +228,8 @@ CrateDB 3.0.5 cluster:
           # It defines the container to run in each pod.
           containers:
           - name: crate
-            # Use the CrateDB 4.2.4 Docker image.
-            image: crate:4.2.4
+            # Use the CrateDB 5.1.1 Docker image.
+            image: crate: 5.1.1
```
Comment (amotl) [inline docs/containers/kubernetes/kubernetes.rst]: There is one space too much.
```suggestion
            image: crate:5.1.1
```

**Review by amotl — CHANGES_REQUESTED** [review]
Thank you Karyn. I've discovered a minor typo, can you address it?

**Review by amotl — APPROVED** [review]
Thanks! 👍

**Pull request merged**